pkgs/by-name/mi/minivmac/package.nix

```diff
@@ -0,0 +1,53 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  buildPackages,
+  alsa-lib,
+  xorg,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "minivmac";
+  version = "2024.06.08";
+
+  src = fetchFromGitHub {
+    owner = "erichelgeson";
+    repo = "minivmac";
+    rev = version;
+    hash = "sha256-iiEE0ioXh3keKUPfAB+YcE1iggGLpxT9N8LSVFDhttE=";
+  };
+
+  buildInputs = [ xorg.libX11 ];
+
+  makeFlags = [ "PREFIX=${placeholder "out"}" ];
+
+  configurePhase = ''
+    ${buildPackages.stdenv.cc}/bin/cc setup/tool.c -o setup_t
+    ./setup_t -t lx64 > setup.sh
+
+    # Patch hardcoded references in setup.sh to cross-aware counterparts
+    sed -i -e "s|gcc |${stdenv.cc.targetPrefix}cc |" setup.sh
```

Comment:
nits: if you're not relying on regexps you might as well use substituteInPlace instead

Reply:
Done